pkg/tools/etcd_helper.go

```diff
@@ -55,12 +56,32 @@ var (
 				"because two concurrent threads can miss the cache and generate the same entry twice.",
 		},
 	)
+	etcdRequestLatenciesSummary = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
+			Name: "etcd_request_latencies_summary",
+			Help: "Etcd request latency summary in microseconds for each operation and object type.",
+		},
+		[]string{"operation", "type"},
+	)
 )
 
+func getTypeNamefromObject(obj runtime.Object) string {
+	return reflect.TypeOf(obj).String()
+}
+
+func getTypeNamefromObjectSlice(objSlice interface{}) string {
+	return reflect.TypeOf(objSlice).String()
```

Comment:
The implementation here is exactly the same as for getTypeNameFromObject. So why do we need both?
Or maybe it should be "reflect.TypeOf(objSlice).Elem().String()"?